AntistasiOfficial.Altis/establishCamp.sqf

```diff
@@ -50,25 +50,27 @@ _pos = position _road findEmptyPosition [1,30,guer_veh_truck];
 
 _vehicle=[_pos, 0,guer_veh_truck, side_blue] call bis_fnc_spawnvehicle;
 _camion = _vehicle select 0;
-_grupo = _vehicle select 2;
+_camion deleteVehicleCrew driver _camion;
 
-_grupoTemp = [getMarkerPos guer_respawn, side_blue, ([guer_grp_sniper, "guer"] call AS_fnc_pickGroup)] call BIS_Fnc_spawnGroup;
-_grupoTemp setGroupId ["Watch"];
+_grupo = [getMarkerPos guer_respawn, side_blue, ([guer_grp_sniper, "guer"] call AS_fnc_pickGroup)] call BIS_Fnc_spawnGroup;
+_grupo setGroupId ["Camp"];
 {
-	_x moveInCargo _camion;
-	[_x] joinSilent _grupo;
-} forEach units _grupoTemp;
+	if (_forEachIndex == 0) then {
+		_x moveInDriver _camion;
+	} else {
+		_x moveInCargo _camion;
+	};
+} forEach units _grupo;
 {[_x] call AS_fnc_initialiseFIAUnit;} forEach units _grupo;
 
+[_grupo, driver _camion] remoteExec ["selectLeader", groupOwner _grupo];
```

Comment:
Code is already run inside the Server, server created-owns that group, remote exec is overkill

Reply:
@StefArma 
BTW such creation script (in general) is HC-unfriendly - ` if (!isServer) exitWith {};` at the start

Reply:
The driver is not the leader of the group as expected

Reply:
I'll have a look this won't mess up with the blufor version. Because the driver is acutally the leader in blufor as far as i remember